pubsub/include/common.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2024 General Motors GTO LLC
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// SPDX-FileType: SOURCE
+// SPDX-FileCopyrightText: 2024 General Motors GTO LLC
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef PUBSUB_COMMON_H
+#define PUBSUB_COMMON_H
+
+#include <up-core-api/uri.pb.h>
+#include <up-cpp/uri/builder/BuildUUri.h>
+#include <up-cpp/uri/builder/BuildUAuthority.h>
+#include <up-cpp/uri/builder/BuildEntity.h>
+#include <up-cpp/uri/builder/BuildUResource.h>
+
+uprotocol::v1::UUri getUri(
+        std::string_view const entName, int const entId, int const entMajorVers,
+        std::string_view const resName, int const resId) {
+
+    using namespace uprotocol::uri;
+
+    return BuildUUri()
+        .setAutority(BuildUAuthority().build())
+        .setEntity(BuildUEntity()
+                .setName(static_cast<std::string>(entName))
+                .setMajorVersion(entMajorVers)
+                .setId(entId)
+                .build())
+        .setResource(BuildUResource()
+                .setName(static_cast<std::string>(resName))
+                .setID(resId)
+                .build())
+        .build();
+}
+
+constexpr std::string_view ENTITY_NAME{"test.app"};
+constexpr int ENTITY_ID{2};
```

Comment:
I might have missed something wrt ENTITY_ID in some of our chats, but what is it supposed to do, and why is it 2 ?

Reply:
Let me create an issue for discussion #11

Reply:
@billpittman - Entity ID is a part of the URI that is a numeric way to locate a uEntity in the system. The broad strokes are covered in the [UUri spec](https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/uri.adoc). In the case of these examples, these values are basically arbitrary. Real systems will use some mechanism to perform a lookup to get these IDs in the future (I don't recall the details off the top of my head).

@evshary - I agree that we should align the examples. Is it reasonable to move forward with this PR and follow up after we've discussed in #11?